test/e2e/tests/ppom-blockaid-alert-metrics.spec.js

```diff
@@ -283,10 +284,7 @@ describe('Confirmation Security Alert - Blockaid @no-mmi', function () {
 
         // Wait for confirmation pop-up
         let windowHandles = await driver.waitUntilXWindowHandles(3);
-        await driver.switchToWindowWithTitle(
-          WINDOW_TITLES.Notification,
-          windowHandles,
-        );
+        await switchToNotificationWindow(driver, 3);
```

Comment:
This is a misuse of the function `switchToNotificationWindow`, which looks like this:
```
async function switchToNotificationWindow(driver, numHandles = 3) {
  const windowHandles = await driver.waitUntilXWindowHandles(numHandles);

  await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog, windowHandles);
}
```
Therefore, you are calling `await driver.waitUntilXWindowHandles(3)` twice.